util/settings/settings.go

```diff
@@ -388,13 +388,13 @@ func (mgr *SettingsManager) notifySubscribers() {
 	}
 }
 
-func readAndConfirmPassword() string {
+func ReadAndConfirmPassword(types string) string {
 	for {
-		fmt.Print("*** Enter an admin password: ")
+		fmt.Printf("*** Enter an %s password: ", types)
```

Comment:
Lets just remove types argument and say "Enter password:" and "Confirm password"